ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml

```diff
@@ -451,6 +451,57 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
no need to always run `nfd-image` (field is automatically set to `always_run: true`, we need to double check it each time new tests are added :))